functions/adoptopenjdk-java.bash

```diff
@@ -0,0 +1,22 @@
+adoptopenjdk_install_apt(){
+  local adoptKey="/tmp/adoptopenjdk.asc"
+  if ! dpkg -s 'adoptopenjdk-8-hotspot-jre' >/dev/null 2>&1; then # Check if already is installed
+    cond_redirect systemctl stop openhab2.service
+    echo -n "$(timestamp) [openHABian] Installing AdoptOpenJDK... "
+    cond_redirect update-alternatives --remove-all java
```

Comment:
put the key fetch first
Else if that fails we would be left without java
move oh start also after key fetch